action.yml

```diff
@@ -20,22 +19,13 @@ runs:
         python3 -m pip install --user pipx
         python3 -m pipx ensurepath
       shell: bash
-    - if: ${{ inputs.poetry-version == 'latest' }}
-      run: |
-        pipx install poetry
-      shell: bash
-    - if: ${{ inputs.poetry-version == 'main' }}
-      run: |
-        pipx install git+https://github.com/python-poetry/poetry.git@main
-      shell: bash
-    - if: ${{ inputs.poetry-version != 'latest' && inputs.poetry-version != 'main' }}
-      run: |
```

Comment:
We do not allow floating versions of packages.
We force to always specify a version.